lib/msf/ui/console/command_dispatcher/dns.rb

```diff
@@ -654,11 +656,7 @@ def print_dns
     Rex::Proto::DNS::UpstreamResolver::Type::SYSTEM.to_s.downcase
   ].freeze
 
-  # XXX: By default rex-text tables strip preceding whitespace:
-  #   https://github.com/rapid7/rex-text/blob/1a7b63993
-  # ca62fd9102665d6986f918ae42cae244e/lib/rex/text/table.rb#L221-L222
-  #   So use https://en.wikipedia.org/wiki/Non-breaking_space as a workaround for now. A change should exist in Rex-Text to support this requirement
-  TABLE_INDENT = "\xc2\xa0\xc2\xa0\\_ ".freeze
+  TABLE_INDENT = "  \\_ ".freeze
```

Comment:
Maybe in the future this should be moved into a centralized place if we want consistent indentation of rows, or maybe it should be part of the 'Styler' concept that we have for rows - which currently adds coloring support and arrows like `=>`, so visual indiciators would work here too